src/interface/metadata.hpp

```diff
@@ -196,7 +196,7 @@ class Metadata {
 
   // 4 constructors, this is the general constructor called by all other constructors, so
   // we do some sanity checks here
-  Metadata(const std::vector<MetadataFlag> &bits, const std::vector<int> &shape = {1},
+  Metadata(const std::vector<MetadataFlag> &bits, const std::vector<int> &shape = {},
```

Comment:
The default value of `shape` being empty distinguishes a scalar field from a vector field with only one entry. Hopefully this isn't too much of a breaking change anywhere downstream, but I think it is necessary for variable output shape to match parthenon variable shape without having e.g. `[nblock, 1, 1, 1, nk, nj, ni]` for all scalar variables.

Reply:
This should probably be commented somewhere in the metadata file, because it's pretty non-intuitive without a comment.

Reply:
I actually think this change is more intuitive, in the sense that why do scalar fields have `shape = {1}` instead of `shape = {1, 1,}` or `shape = {1, 1, 1}`? That being said, this is a change to the way things have been for quite a while so I agree I should add a comment here

Reply:
I don't disagree regarding shape from a physics perspective, but from an array perspective I think the original way makes as much intuitive sense to me as anything.

To be clear, I don't object to your fix. Just want to make sure it's documented.